### PR TITLE
ENG-348 |  Update api docs for SUPP-12 price_breakdown_with_passenger_type_code

### PR DESCRIPTION
**Pull request by rob-dumont**
Added in the price_breakdown_with_passenger_type_code array of objects to search response based on changes from SUPP-12

## LOCALQA APPROVED BY
@sunny-trip-ninja

### CODE REVIEW

**Comment by mschembri-TN** [comment]
Task linked: [ENG-348 Update API docs for SUPP-12](https://app.clickup.com/t/14197839/ENG-348)

**Comment by mschembri-TN** [comment]
Task linked: [SUPP-12 Fare breakdown per passenger Type.](https://app.clickup.com/t/14197839/SUPP-12)

**Review by sunny-trip-ninja — APPROVED** [review]

File: tn_api_spec.yml [hunk 1]
```diff
@@ -522,6 +522,37 @@ components:
...
 
+    PriceBreakdownWithPassengerTypeCode:
+      type: array
+      description: Displays the pricing for each passenger type
+      items: 
+        type: object
+        properties:
+          passenger_type_code: 
+            type: string
+            example: ADT
```
Comment (sunny-trip-ninja) [inline tn_api_spec.yml]: `"ADT"`?

**Review by sunny-trip-ninja — APPROVED** [review]

**Comment by sunny-trip-ninja** [comment]
localQA approved

**Pull request merged**